packages/shared/src/utils/cfd/cfd.ts

```diff
@@ -328,27 +360,27 @@ export const getAuthenticationStatusInfo = (account_status: GetAccountStatus) =>
         onfido_status,
         manual_status,
         acknowledged_status,
-        poi_acknowledged_for_vanuatu_maltainvest,
-        poi_poa_verified_for_bvi_labuan,
-        poa_acknowledged,
-        poi_poa_verified_for_vanuatu_maltainvest,
+        poi_acknowledged_for_maltainvest,
+        poi_poa_verified_for_bvi_labuan_vanuatu,
+        poa_acknowledged: acknowledged_status.includes(poa_status),
```

Comment:
can we declare this variable along with others since rest are declared there.

Reply:
Sure, I'll update this